polar_training2tcx

```diff
@@ -173,13 +173,13 @@ def output_tcx(parsed)
                           }
                         }
                       end
-                    }.xmlns = 'http://www.garmin.com/xmlschemas/TrainingCenterDatabase/v2'
```

Comment:
Looks like Polar Flow excludes the XML namespace of the first Trackpoint of each Track only. Strange!